Imply the existence of third party rehype plugins for unsupported math rendrers in README (#111)
remarkjs/remark-math: readme.md

When dealing with markdown, you optionally use `remark-math`, or alternatively
use fenced code (` ```math `).

> Then, you either use `rehype-katex` or `rehype-mathjax` (or one of third party rehype plugins [(see here for major ones)](https://github.com/rehypejs/rehype/blob/main/doc/plugins.md#list-of-plugins) for other math renderers e.g. Temml) to render math in HTML.

Reviewer: ```suggestion
Then, you either use `rehype-katex` or `rehype-mathjax` to render math in HTML.

There are also other plugins that work with `remark-math`.
Such as [`rehype-mathml`](https://github.com/Daiji256/rehype-mathml)
which uses [`ronkok/Temml`](https://github.com/ronkok/Temml).
```

How about this?

You specifically want to call out related projects.
Specifically one related project.

No need to link the list of plugins, which is highlighted in several places already, and I don’t think it needs to be highlighted *from* plugins too.

Author: I will mostly agree. I think the following is better:

```diff
-Such as [`rehype-mathml`](https://github.com/Daiji256/rehype-mathml)
+Such as [`rehype-mathml`](https://github.com/Daiji256/rehype-mathml),
```

Reviewer: I would normally agree, but I use commas myself too much!
I think that’s a comma splice: https://en.wikipedia.org/wiki/Comma_splice

Author: Sorry for lacking the context but I mean https://www.scribbr.com/commas/comma-before-which/.

Reviewer: Hmm. From that source I do not draw that a comma should be used here. I instead draw the inverse. I see a restrictive clause?
